<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Grafy 1 </string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="GraphSizeWidget" name="graphSize" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolBox" name="toolBox">
         <property name="currentIndex">
          <number>8</number>
         </property>
         <widget class="QWidget" name="page">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>330</width>
            <height>190</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Macierz sąsiedztwa</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>M=</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="MatrixView" name="matrix" native="true">
               <property name="sizePolicy">
                <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>330</width>
            <height>190</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Lista sąsiedztwa</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QScrollArea" name="scrollArea">
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>310</width>
                <height>154</height>
               </rect>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="ListView" name="list" native="true"/>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="AddConnection" name="connectionAdd" native="true">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>330</width>
            <height>190</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Generatory</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="GeneratorsWidget" name="generators" native="true">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_5">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>330</width>
            <height>190</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Macierz incydencji</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QScrollArea" name="scrollArea_2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents_2">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>310</width>
                <height>170</height>
               </rect>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="ConnectionMatrixView" name="connectionMatrix" native="true"/>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_6">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>330</width>
            <height>190</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Sekwencja graficzna i graf k-regularny</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="sequenceWidget" name="widget_2" native="true"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_7">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>330</width>
            <height>190</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Randomizacja grafu</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="RandomizerWidget" name="widget_3" native="true"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_8">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>330</width>
            <height>190</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Cykl Hamiltona</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="HamiltonianWidget" name="widget_4" native="true"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_9">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>330</width>
            <height>190</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Cykl Eulera</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="EulerianCircuitWidget" name="widget_5" native="true"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_10">
          <attribute name="label">
           <string>Spójna składowa</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="ConsistentWidget" name="widget_6" native="true"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_4">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>330</width>
            <height>190</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Obsługa plików</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="FileManagerWidget" name="widget" native="true"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="GraphPainter" name="graph" native="true">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>300</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionDfs">
   <property name="text">
    <string>dfs</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GraphPainter</class>
   <extends>QWidget</extends>
   <header>graphpainter.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MatrixView</class>
   <extends>QWidget</extends>
   <header>matrixview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GraphSizeWidget</class>
   <extends>QWidget</extends>
   <header>graphsizewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ListView</class>
   <extends>QWidget</extends>
   <header>listview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AddConnection</class>
   <extends>QWidget</extends>
   <header>addconnection.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GeneratorsWidget</class>
   <extends>QWidget</extends>
   <header>generatorswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ConnectionMatrixView</class>
   <extends>QWidget</extends>
   <header>connectionmatrixview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FileManagerWidget</class>
   <extends>QWidget</extends>
   <header>filemanagerwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>sequenceWidget</class>
   <extends>QWidget</extends>
   <header>sequencewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RandomizerWidget</class>
   <extends>QWidget</extends>
   <header>randomizerwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>HamiltonianWidget</class>
   <extends>QWidget</extends>
   <header>hamiltonianwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>EulerianCircuitWidget</class>
   <extends>QWidget</extends>
   <header>euleriancircuitwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ConsistentWidget</class>
   <extends>QWidget</extends>
   <header>consistentwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
